plaid/deposit_switch.go

```diff
@@ -0,0 +1,115 @@
+package plaid
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+type getDepositSwitchRequest struct {
+	ClientID        string `json:"client_id"`
+	Secret          string `json:"secret"`
+	DepositSwitchID string `json:"deposit_switch_id"`
+}
+
+// GetDepositSwitchResponse details the response from /deposit_switch/get.
+type GetDepositSwitchResponse struct {
+	DepositSwitchID               string  `json:"deposit_switch_id"`
+	TargetItemID                  string  `json:"target_item_id"`
+	TargetAccountID               string  `json:"target_account_id"`
+	State                         string  `json:"state"`
+	RequestID                     string  `json:"request_id"`
+	DateCreated                   string  `json:"date_created,omitempty"`
+	IsAllocatedRemainder          bool    `json:"is_allocated_remainder,omitempty"`
+	AccountHasMultipleAllocations bool    `json:"account_has_multiple_allocations,omitempty"`
+	PercentAllocated              float64 `json:"percent_allocated,omitempty"`
+	AmountAllocated               float64 `json:"amount_allocated,omitempty"`
```

Comment:
Hmmm....These aren't implemented yet - I would avoid putting them into the Client docs until they are.

Reply:
⚡️